creme/creme_core/static/creme_core/js/detailview.js

```diff
@@ -77,13 +77,19 @@
 
         'creme_core-hatmenubar-update': function(url, options, data) {
             return this._postQueryAction(url, options, data);
+        },
+
+        'creme_core-hatmenubar-delete': function(url, options, data) {
```

Comment:
In brick actions, there is a "update-redirect" action => should we copy this?
 - more consistent (same name between different "systems")
 - this action is not tied to deletion; I presume it's less confusing to delete with an "update-redirect" action, than use "delete" for another action which does not perform a deletion.